docs/en/sql-reference/functions/string-functions.md

```diff
@@ -1196,3 +1196,39 @@ Result:
 Same as concatWithSeparator, the difference is that you need to ensure that concatWithSeparator(sep, expr1, expr2, expr3...) → result is injective, it will be used for optimization of GROUP BY.
 
 The function is named “injective” if it always returns different result for different values of arguments. In other words: different arguments never yield identical result.
+
+## soundex
+
+Returns soundex code of a string.
```

Comment:
A reference in the docs to Wikipedia or some other explanation would be nice.

Reply:
Thanks for your comment, I added Wikipedia's URL in the docs.